libraries/src/Toolbar/Toolbar.php

```diff
@@ -70,7 +70,7 @@ class Toolbar
 	/**
 	 * Stores the singleton instances of various toolbar.
 	 *
-	 * @var    Toolbar
+	 * @var    array
```

Comment:
```suggestion
	 * @var    Toolbar[]
```

Not sure if we use this syntax

Reply:
We do